app/views/admin/home/index.html.erb

```diff
@@ -14,11 +14,29 @@
   </p>
 
   <div class="mt-4">
-    Vous possèdez les rôles suivants :
+    Vous possèdez les rôles/accès suivants :
     <% if current_user.roles.any? %>
-      <ul>
-        <% current_user.roles.each do |role| %>
-          <li><%= role.name %></li>
+      <ul class="mt-2">
```

Comment:
We can extract this in a view helper that takes a user as a parameter. This allows us not to duplicate the code between this view and the next view.